CMakeLists.txt

```diff
@@ -571,6 +576,15 @@ if(NOT AVIF_CODEC_AOM_ENABLED
     message(WARNING "libavif: No decoding library is enabled.")
 endif()
 
+if(AVIF_BUILD_APPS
```

Comment:
IMPORTANT: Keep line 151 with this if statement. Alternatively, use an if-else statement.

Reply:
No, because `AVIF_USE_CXX` is set to ON above (e.g. with Compliance Warden)

Reply:
You are right. Thanks.